mj_envs/logger/roboset_logger.py

```diff
@@ -51,7 +51,7 @@ def path2dataset(self, path:dict, config_path=None)->dict:
             dataset['config'] = config
 
         if 'user_cmt' in path.keys():
-            dataset['config/solved'] = float(path['user_cmt'])
+            dataset['config/solved'] = np.array(np.float16(path['user_cmt']))
```

Comment:
Can you change it to something along the lines of  
`np.array(path['user_cmt'], dtype=np.float16)`